packages/dds/tree/src/forest/cursor.ts

```diff
@@ -17,6 +17,16 @@ export const enum TreeNavigationResult {
     Ok = 1,
 }
 
+/**
+ * Value stored on a node.
+ *
+ * TODO: `Serializable` is not really the right type to use here,
+ * since may types (including functions) are "Serializable" (according to the type) despite not being serializable.
+ *
+ * Use this type instead of directly using Serializable for both clarity and so the above TODO can be addressed.
+ */
+export type Value = undefined | Serializable;
```

Comment:
Nit: Maybe 'TreeValue' as 'Value' is a pretty generic name for a public export?

Reply:
I considered "TreeValue". Someone might think it is a value that is a tree: I don't like that ambiguity.
That said, "Value" isn't much better, but aligns with the pattern we have fore trees: TreeSchema and Tree, ValueSchema and Value.